apps/dashboard/app/(app)/audit/actions.ts

```diff
@@ -2,14 +2,14 @@ import { db } from "@/lib/db";
 import { clerkClient } from "@clerk/nextjs";
 import { redirect } from "next/navigation";
 
-export const getWorkspace = async (tenantId: string) => {
+export const getWorkspace = async (orgId: string) => {
   try {
     let members = null;
 
-    if (!tenantId.startsWith("user_")) {
+    if (!orgId.startsWith("user_")) {
```

Comment:
Do we actually need this? No workspace will have a tenant/org Id with user_ prefix, even if we merge.

This looks like a new page that was added after the initial new auth work. I'll put this on my TODO list to revamp this for new auth.

Reply:
yeah that's dumb, my bad